### ISSUE TITLE
Feature request: Starred/ Pinned experiments

### ISSUE THREAD

**Issue by majdzr** [allegroai/trains#169]
Thanks again for this great product. 
Is there a way to star favorite experiments or to short list some or even to pin to top? I find myself scrolling down in the list to reach _the_ experiment I want to compare to. 

Thanks again!

**Comment by ainoam** [comment]
Hi @majdzr ,

For creating a multi-experiment comparison - once you select (mark the checkbox) an experiment, the selection holds regardless of the current view, so you can use column filters or the top search bar to quickly get to the additional experiments you want to select (as long as they're all within the same project scope).

Or, does your use case go beyond selecting experiments such as any of the following:
- "Forcing" a set of experiments to be shown in adjacent rows regardless of the rest of the table order?
- Creating a custom "quick filter" to easily return to the same set of experiments (Note you can use such a similar "Star" functionality by applying your own custom tag to the experiments of interest and then filtering on it).

What makes the most sense to you?

**Comment by majdzr** [comment]
Hey, thanks for the quick reply.
Indeed, I'm already using this, but I also think these two options might
even make the process easier. Or even a simpler solution of creating a
starred/master experiment(s) that always show in the comparison, regardless
of the filter or being selected.

What do you think?

**Comment by ainoam** [comment]
@majdzr So it sounds like we're discussing several possible features - From your latest comment I understand that your compelling need is for a comparison template (notice I'm distinguishing between what's seen in the experiment table, and the members in the comparison) to which you can repeatedly add additional experiments?

Notice, that within the existing capabilities, you can have your browser bookmark a specific comparison (or, use the "share" button, to get the comparison URL), and then start of from this saved comparison and use "add experiment" for additional experiments.

**Comment by majdzr** [comment]
@ainoam Yes, you can see it as a comparison template, containing several ("pinned"/"starred") experiments, that can be removed (unpinned or unstarred) easily.

**Comment by patrickdasilva-clearml** [comment]
Hi @majdzr , we're closing this issue as stale since it's been created too long ago. If you think the issue is still relevant, please make sure the issue persists on a recent version of ClearML (ideally the latest) before re-opening the issue. We'll gladly look at it again.
